leasing/importer/area.py

```diff
@@ -422,8 +421,10 @@ def get_update_data(
             "metadata": metadata,
         }
 
-        if area_import["area_type"] == AreaType.PLAN_UNIT:
-            update_data["external_id"] = row.id if row.id else None
+        ext_id = getattr(row, "id", None)
+
+        if ext_id:
```

Comment:
Did you mean `if ext_id is None:`?

Reply:
Just for clarification, ext_id could perhaps be `0`.
```
>>> bool(0)
False
```

Reply:
No, `ext_id` is taken from `row.id`. If it is defined, then `update_data` will have `external_id` key and `ext_id` assigned to it, and `external_id` will be updated. If the row does not have id, then `update_data` will not have `external_id` key.

Reply:
It should probably be `if ext_id is not None`

Reply:
> No, `ext_id` is taken from `row.id`. If it is defined, then `update_data` will have `external_id` key and `ext_id` assigned to it, and `external_id` will be updated. If the row does not have id, then `update_data` will not have `external_id` key.

I meant that if `row.id` is `0`, then it will be falsy, but it might be completely valid case to have id 0. If `ext_id = 0` then `bool(ext_id) -> False`.